payjoin-cli/src/app/v2.rs

```diff
@@ -306,57 +305,21 @@ impl App {
     }
 }
 
-async fn unwrap_ohttp_keys_or_else_fetch(config: &AppConfig) -> Result<payjoin::OhttpKeys> {
+async fn unwrap_ohttp_keys_or_else_fetch(config: AppConfig) -> Result<payjoin::OhttpKeys> {
     if let Some(keys) = config.ohttp_keys.clone() {
         Ok(keys)
     } else {
-        fetch_ohttp_keys(&config.ohttp_relay, &config.pj_endpoint).await
+        Ok(tokio::task::spawn_blocking(move || {
+            payjoin_defaults::fetch_ohttp_keys(
+                config.ohttp_relay.clone(),
+                config.pj_endpoint.clone(),
+            )
+        })
```

Comment:
the `danger-local-https` feature from `payjoin-default`  is not used by the `cli` and its not included in the cli `cargo.toml`.  should we still accommodate for that?

Reply:
Yes, it should be used if is danger-local-https && v2 features are set. It can probably be switched on by the payjoin-cli/danger-local-https feature since it's a test feature and an extra dependency shouldn't hurt there.